src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -72,6 +72,8 @@ bool GcsActor::IsDetached() const { return actor_table_data_.is_detached(); }
 
 std::string GcsActor::GetName() const { return actor_table_data_.name(); }
 
+std::string GcsActor::GetNamespace() const { return actor_table_data_.ray_namespace(); }
```

Comment:
should be consistent in naming -- either just call it "namespace" or "ray namespace" everywhere. Preference for the former.

Reply:
I have a preference for `namespace` too, but it's a reserved keyword in C++.